pandas/io/pytables.py

```diff
@@ -1816,10 +1816,14 @@ class GenericIndexCol(IndexCol):
     def is_indexed(self):
         return False
 
-    def convert(self, values, nan_rep, encoding, errors):
+    def convert(self, values, nan_rep, encoding, errors, **kwargs):
         """ set the values from this selection: take = take ownership """
```

Comment:
doc-string here

Reply:
:+1: adding a docstring explaining the `start` and stop` parameters, commit upcoming